app/controllers/admin/profiles_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Admin
+  class ProfilesController < ModuleController
+
+    def index
+      @profiles = Profile.all
+      @profiles = @profiles.order(state: params[:filter] == 'state_desc' ? :asc : :desc) if params[:filter].present?
```

Comment:
we don't need an order by state but a filter by state to show for so use a where